### ISSUE TITLE
Object Type in KUs not set

### ISSUE THREAD

**Issue by cswertz** [cswertz/lerndorf#126]
The object type in KUs is not set. It should be set to the chosen content tool (currently, just "editor", but "forum", "task" or "H5P" will be there in the future).

**Comment by stylesuxx** [comment]
Yes, this is already adapted in my local branch containing the "Forschungsinterface".

**Comment by stylesuxx** [comment]
@cswertz I was looking at the wrong field I think. I was refering to one of the LogUser fields, but that was not what you meant. As far as I can see the taxonomy for object type is missing, no?

**Comment by cswertz** [comment]
Right. I added the object type and added "editor" (the only one we have yet). If the editor is used, "editor" should be set.
